partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -78,10 +82,17 @@
   public static final String TIMEZONE_DEFAULT = "";
   public static final String TIMEZONE_DISPLAY = "Timezone";
 
-  // Schema group
-  public static final String SCHEMA_GENERATOR_CLASS_CONFIG = "schema.generator.class";
-  public static final String SCHEMA_GENERATOR_CLASS_DOC = "The schema generator to use with partitioners.";
-  public static final String SCHEMA_GENERATOR_CLASS_DISPLAY = "Schema Generator Class";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_CONFIG = "timestamp.extractor";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_DOC =
+      "The extractor used to get timestamp for records when partitioning with ``TimeBasedPartitioner``.";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_DEFAULT = "Wallclock";
```

Comment:
Ok, I see now that Wallclock, Record, and RecordField are special cases. Add to the doc string? Otherwise I wouldn't have even thought to try the other two.